apps/oc_chef_wm/priv/dispatch.conf

```diff
@@ -125,4 +125,7 @@
 {["system_recovery"], oc_chef_wm_system_recovery, []}.
 {["license"], oc_chef_wm_license, []}.
 
+{["organizations", organization_id, "policies"], oc_chef_wm_policies, []}.
+{["organizations", organization_id, "policies", policy_group, policy_name], oc_chef_wm_named_policy, []}.
```

Comment:
Would we need , for example,  "GET $PREFIX/policies/mygroup" to get policies within a group?